src/renderers/dom/client/ReactDOMTreeTraversal.js

```diff
@@ -80,13 +80,18 @@ function getParentInstance(inst) {
 }
 
 /**
- * Simulates the traversal of a two-phase, capture/bubble event dispatch.
+ * Simulates the traversal of a two-phase, capture/bubble event dispatch
+ * with skipping disabled instances.
  */
 function traverseTwoPhase(inst, fn, arg) {
   var path = [];
   while (inst) {
     path.push(inst);
-    inst = inst._hostParent;
+    var parent = getParentInstance(inst);
+    while (parent && parent._currentElement && parent._currentElement.props.disabled) {
```

Comment:
Yeah, also I figured, that here https://github.com/facebook/react/blob/master/src/renderers/dom/client/eventPlugins/SimpleEventPlugin.js#L152 we already has this logic, but I do not know good place to move it, so we can reuse it, any ideas?